### ISSUE TITLE
How do you run a server with srt-live-transmit?

### ISSUE THREAD

**Issue by remino** [Haivision/srt#2049]
I was using RTMP before to transmit video from my iPhone using Larix Broadcaster to my Raspberry Pi. Then, OBS on a separate computer would watch that RTMP feed, which it would in turn transmit it, with overlays on top, to another RTMP server.

I’m trying to use SRT instead to lessen the data load, and allow HEVC video to be used instead.

Is it possible to run a SRT server with srt-live-transmit which would listen on a port, to which my Larix Broadcaster can push SRT HEVC video, then for OBS to pull a live SRT feed from it?

I tried:

```
srt-live-transmit srt://:8080 srt://:8081 -v
```

But when Larix connects to port 8080 to send a SRT stream, all I see in the console on my server is the following:

```
Media path: 'srt://:8080' --> 'srt://:8081'                                                                                                          
Opening SRT source listener on :8080                                                                                                                 
Binding a server on :8080 ...
 listen...
Opening SRT target listener on :8081
Binding a server on :8081 ...
 listen...
 accept...
 connected.
Accepted SRT source connection
2632 bytes lost, 0 bytes sent, 2632 bytes received
2826768 bytes lost, 0 bytes sent, 2826768 bytes received
3484768 bytes lost, 0 bytes sent, 3484768 bytes received
5187672 bytes lost, 0 bytes sent, 5187672 bytes received
6147036 bytes lost, 0 bytes sent, 6147036 bytes received
6920844 bytes lost, 0 bytes sent, 6920844 bytes received
7742028 bytes lost, 0 bytes sent, 7742028 bytes received
8855364 bytes lost, 0 bytes sent, 8855364 bytes received
10052924 bytes lost, 0 bytes sent, 10052924 bytes received
11607120 bytes lost, 0 bytes sent, 11607120 bytes received
12958652 bytes lost, 0 bytes sent, 12958652 bytes received
14833952 bytes lost, 0 bytes sent, 14833952 bytes received
…
```

In other words, all the packets get lost. Not sure how to work around that.

**Comment by mkdryden** [comment]
Larix is connected and is sending video, but no target is connected, so srt-live-transmit is simply dropping everything because it has nowhere to send it. Once OBS is connected, it will stop losing them.
